### DISCUSSION TITLE
Welcome to Discord-Clone Discussions!

### DISCUSSION THREAD

**Discussion by Yeasir0032** [Yeasir0032/Discord-Clone#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by venkat2295** [comment]
you have already completed the whole website i want to help if possible but i am unable to find any place where i can contruibute

**Reply by Yeasir0032** [reply]
Okay I am giving a small **issue** lets do it.
A website is never **completed**, if that was case then we might not need the maintainers.
You just have to find something in the application that is not pleasing to you. Like 

- Performance
- Security 
- Loading screen 
- Smooth scrolling
- API handling

**Reply by Yeasir0032** [reply]
Check this [issue](https://github.com/Yeasir0032/Discord-Clone/issues/26) @Venkatknight2295
Ask assign request in the issue
